bridge/config/config.go

```diff
@@ -85,91 +85,94 @@ type ChannelMember struct {
-	NickServNick           string     // IRC
-	NickServUsername       string     // IRC
-	NickServPassword       string     // IRC
-	NicksPerRow            int        // mattermost, slack
-	NoHomeServerSuffix     bool       // matrix
-	NoSendJoinPart         bool       // all protocols
-	NoTLS                  bool       // mattermost, xmpp
-	Password               string     // IRC,mattermost,XMPP,matrix
-	PrefixMessagesWithNick bool       // mattemost, slack
-	PreserveThreading      bool       // slack
-	Protocol               string     // all protocols
-	QuoteDisable           bool       // telegram
-	QuoteFormat            string     // telegram
-	QuoteLengthLimit       int        // telegram
-	RejoinDelay            int        // IRC
-	ReplaceMessages        [][]string // all protocols
-	ReplaceNicks           [][]string // all protocols
-	RemoteNickFormat       string     // all protocols
-	RunCommands            []string   // IRC
-	Server                 string     // IRC,mattermost,XMPP,discord
-	SessionFile            string     // msteams,whatsapp
-	ShowJoinPart           bool       // all protocols
-	ShowTopicChange        bool       // slack
-	ShowUserTyping         bool       // slack
-	ShowEmbeds             bool       // discord
-	SkipTLSVerify          bool       // IRC, mattermost
-	SkipVersionCheck       bool       // mattermost
-	StripNick              bool       // all protocols
-	StripMarkdown          bool       // irc
-	SyncTopic              bool       // slack
-	TengoModifyMessage     string     // general
-	Team                   string     // mattermost, keybase
-	TeamID                 string     // msteams
-	TenantID               string     // msteams
-	Token                  string     // gitter, slack, discord, api
-	Topic                  string     // zulip
-	URL                    string     // mattermost, slack // DEPRECATED
-	UseAPI                 bool       // mattermost, slack
-	UseLocalAvatar         []string   // discord
-	UseSASL                bool       // IRC
-	UseTLS                 bool       // IRC
-	UseDiscriminator       bool       // discord
-	UseFirstName           bool       // telegram
-	UseUserName            bool       // discord, matrix
-	UseInsecureURL         bool       // telegram
-	VerboseJoinPart        bool       // IRC
-	WebhookBindAddress     string     // mattermost, slack
-	WebhookURL             string     // mattermost, slack
+	AuthCode                string // steam
+	BindAddress             string // mattermost, slack // DEPRECATED
+	Buffer                  int    // api
+	Charset                 string // irc
+	ClientID                string // msteams
+	ColorNicks              bool   // only irc for now
+	Debug                   bool   // general
+	DebugLevel              int    // only for irc now
+	DisablePingEveryoneHere bool   // discord
+	DisablePingRoles        bool   // discord
+	DisablePingUsers        bool   // discord
```

Comment:
I think it would better to keep this in line with the documentation's terminology, and invert the value accordingly:

```suggestion
	ParseEveryoneMentions bool   // discord
	ParseRoleMentions        bool   // discord
	ParseUserMentions        bool   // discord
```

Reply:
That's what I did first, but I then inverted it so that the default bool value (false) reflect the current behavior (all allowed), for compatibility.
I guess I can get it working with config.IsKeySet()

Reply:
Resolved by https://github.com/42wim/matterbridge/pull/1462/commits/cb6d9598ed78c5cae77bcc10683ce50d4a0e60e3